src/utilities/issue-embed-generator.ts

```diff
@@ -0,0 +1,36 @@
+import { MessageEmbed } from 'discord.js';
+import { IssueData } from '../APIs/issue-def';
+
+export function issueEmbedGenerator(issueData: IssueData): MessageEmbed {
+  const {
+    title,
+    user,
+    comments,
+    state,
+    html_url,
+    pull_request,
+    number
+  } = issueData;
+  const bannerImageLink =
```

Comment:
Is this the best/most stable way to get this file? This seems like one of those "magic hard coded strings" thru github user content 🤔

Reply:
I'm not seeing where it's sent down in the API, tho. 🤔